test/setup.ts

```diff
@@ -0,0 +1,8 @@
+import { toContainGraph } from './custom-matchers/toContainGraph'
+import { toEqualGraph } from './custom-matchers/toEqualGraph'
+
+// adding custom matchers
+expect.extend({
+  toContainGraph,
+  toEqualGraph
+})
```

Comment:
OK to add this file, but please call it from individual test files, not from jest config.

Reply:
I'm uncertain of how much value it is to make this opt-in? Yes, there's the problem of discoverability, perhaps I should add a section to developer docs describing these custom matchers instead?

Reply:
See https://github.com/solid/solid-ui/pull/285

Reply:
Hence my point on discoverability. I don't think we should make custom matchers opt-in though, they should always be available to all tests.

Reply:
We're commenting on a commit that doesn't exist anymore? My original comment was about silencing the debug messages, I didn't intend to say anything about custom matchers.

Sounds like yeah, these matchers should always be available to all tests.

Reply:
Then you commented the wrong place ;) Or maybe we're seeing different things:

<img width="653" alt="Screen Shot 2020-03-25 at 17 54 39" src="https://user-images.githubusercontent.com/775139/77563550-b6a17700-6ec1-11ea-93a8-eb79aad7a49c.png">

Ok, but then this PR should be ok, and I'll merge it as is.